src/Plutip/Server.purs

```diff
@@ -162,8 +181,60 @@ startOgmios cfg params = liftEffect $ spawn "ogmios" ogmiosArgs
     , params.nodeConfigPath
     ]
 
-stopOgmios :: ChildProcess -> Aff Unit
-stopOgmios = liftEffect <<< kill SIGINT
+stopChildProcess :: ChildProcess -> Aff Unit
+stopChildProcess = liftEffect <<< kill SIGINT
+
+startPlutipServer :: PlutipConfig -> Aff ChildProcess
+startPlutipServer cfg = do
+  p <- liftEffect $ spawn "plutip-server" [ "-p", UInt.toString cfg.port ]
+    defaultSpawnOptions { detached = true }
+  delay $ Milliseconds 100.0
+  pure p
+
+startPostgresServer
+  :: PostgresConfig -> ClusterStartupParameters -> Aff ChildProcess
+startPostgresServer pgConfig _ = do
+  randomStr <- liftEffect $ fromJustEff "hello" =<< head <$> randomSample
+    genAlphaString
+  tmpDir <- liftEffect tmpdir
+  let
+    workingDir = tmpDir <> "/" <> randomStr
+    databaseDir = workingDir <> "/postgres/data"
+  liftEffect $ void $ execSync ("initdb " <> databaseDir) defaultExecSyncOptions
```

Comment:
is `initdb` from the same nix package as `psql`?

Reply:
yes, everything is available through `pkgs.postgresql`